modules/touch/include/touchinteraction.h

```diff
@@ -184,11 +177,14 @@ class TouchInteraction : public properties::PropertyOwner {
     properties::FloatProperty _centroidStillThreshold;
     properties::BoolProperty  _panEnabled;
     properties::FloatProperty _interpretPan;
-    properties::FloatProperty _slerpTime;
     properties::Vec4Property _friction;
-    properties::FloatProperty _pickingRadiusMinimum;
     properties::FloatProperty _constTimeDecay_secs;
 
+    properties::BoolProperty _enableDirectManipulation;
+    properties::FloatProperty _directTouchDistanceThreshold;
+    //properties::FloatProperty _orbitSpeedThreshold;
```

Comment:
Is this future work to fix? If so should we add a comment about why they are commented out?

Reply:
Removed. See previous comment